app/epa/settings.py

```diff
@@ -1,250 +1,251 @@
-LOGOUT_REDIRECT_URL = "home_cpn"
-
-CRISPY_TEMPLATE_PACK = "bootstrap4"
-
-# Please note, we don't use Django's internal email system,
-# we implement our own, using exchangelib
-USE_EXCHANGE_EMAIL_BACKEND = ast.literal_eval(os.getenv("USE_EXCHANGE_EMAIL_BACKEND", "True"))
-# The Exchange account which sends emails
-EXCHANGE_ACCOUNT = os.getenv("EXCHANGE_ACCOUNT", "dummy@dummy.com")
-EXCHANGE_PW = os.getenv("EXCHANGE_PW", "dummypw")
-EXCHANGE_EMAIL = os.getenv("EXCHANGE_EMAIL", "dummy@dummy.com")
-EXCHANGE_SERVER = os.getenv("EXCHANGE_SERVER", "dummy.com")
-# Email addresses to which feedback emails will be sent
-RECIPIENTS = os.getenv("RECIPIENTS", "dummy@dummy.com,dummy2@dummy.com").split(",")
-EMAIL_SUBJECT_PREFIX = os.getenv("EMAIL_SUBJECT_PREFIX", "[open_plan] ")
-
-MESSAGE_TAGS = {
-    messages.DEBUG: "alert-info",
-    messages.INFO: "alert-info",
-    messages.SUCCESS: "alert-success",
-    messages.WARNING: "alert-warning",
-    messages.ERROR: "alert-danger",
-}
-
-USE_PROXY = ast.literal_eval(os.getenv("USE_PROXY", "True"))
-PROXY_ADDRESS_LINK = os.getenv("PROXY_ADDRESS", "http://proxy:port")
-PROXY_CONFIG = ({"http://": PROXY_ADDRESS_LINK, "https://": PROXY_ADDRESS_LINK}) if USE_PROXY else ({})
-
-MVS_API_HOST = os.getenv("MVS_API_HOST", "https://mvs-eland.rl-institut.de")
-MVS_POST_URL = f"{MVS_API_HOST}/sendjson/"
-MVS_GET_URL = f"{MVS_API_HOST}/check/"
-MVS_LP_FILE_URL = f"{MVS_API_HOST}/get_lp_file/"
-MVS_SA_POST_URL = f"{MVS_API_HOST}/sendjson/openplan/sensitivity-analysis"
-MVS_SA_GET_URL = f"{MVS_API_HOST}/check-sensitivity-analysis/"
-
-# Allow iframes to show in page
-X_FRAME_OPTIONS = "SAMEORIGIN"
-
-import sys
-
-LOGGING = {
-    "version": 1,
-    "disable_existing_loggers": False,
-    "formatters": {
-        "dtlnm": {
-            "format": "%(asctime)s - %(levelname)8s - %(name)s - %(message)s",
-            "datefmt": "%Y-%m-%d %H:%M:%S",
-        }
-    },
-    "handlers": {
-        "info_file": {
-            "level": "INFO",
-            "class": "logging.FileHandler",
-            "filename": "django_epa_info.log",
-            "formatter": "dtlnm",
-        },
-        "warnings_file": {
-            "level": "WARNING",
-            "class": "logging.FileHandler",
-            "filename": "django_epa_warning.log",
-            "formatter": "dtlnm",
-        },
-        "console": {
-            "level": "WARNING",
-            "class": "logging.StreamHandler",
-            "stream": sys.stdout,
-        },
-    },
-    "loggers": {
-        "": {
-            "handlers": ["info_file", "warnings_file", "console"],
-            "level": "DEBUG",
-            "propagate": True,
-        },
-        "asyncio": {"level": "WARNING"},
-    },
-}
-
-# DJANGO-Q CONFIGURATION
-# source: https://django-q.readthedocs.io/en/latest/configure.html
-Q_CLUSTER = {
-    "name": "django_q_orm",
-    "workers": 4,
-    "timeout": 90,
-    "retry": 120,
-    "queue_limit": 50,
-    "orm": "default",
-}
+"""
```

Comment:
wierd that is considered the whole file changed when only one line of it did ...

Reply:
whoops. not sure what happened there